### PR TITLE
Pqccm add nexus entry

### PR DESCRIPTION
**Pull request by VDNexus**
I would like to submit Nexus Group entry in the PQC compatibility Matrix as we will have a testing service live in the coming month and planned for a production grade version as soon as possible when specifications are official and HSMs offer support.

### CODE REVIEW

File: _index.md [hunk 1]
```diff
@@ -103,6 +104,11 @@ Java and C# APIs with all NIST candidate support, and some older ones. [Availabl
 
 The [Bouncy Castle for kotlin](https://github.com/bcgit/bc-kotlin) open source package provides a script/command line interface for generating certificate chains with different algorithms, including hybrid certificates using X.509 section 9.8 alternative signatures. 
 
+## Nexus Group (IN Groupe)
+PQC online test service available starting April 2024.
+Nexus Group secures society by enabling trusted identities for people and things. Our future-proof PKI helps organizations enable trust with certificate-based identities for citizens, employees, devices, applications, things, and IoT. Our PKI, Certificate Manager is certified Common Criteria (CC) EAL4+ and we deliver it on Premise and as Service.
```
Comment (primetomas) [inline _index.md:109]: I think this text is too salezy, the list should be pure fact based. Look at the other vendor sections for guidance to keep it to the bare facts of the features offered by product itself. The company itself needs no presentation.

File: _index.md [hunk 2]
```diff
@@ -67,6 +67,7 @@ The table lists information from vendors related to support for Post Quantum Cry
...
 | [Keyfactor](#keyfactor)                 | EJBCA            | PKI              | 2023-11-20   |                  :x:                  |                :clock1:          |        :x:         |         :x:         |  :heavy_check_mark:   |    :heavy_check_mark:    |           :x:           |         :x:          |         :x:         |           :x:           |        :x:         |
 | [MTG AG](#mtg-ag)                       | Corporate PKI    | PKI              | 2023-09-25   |                  :x:                  |                :x:                 |        :x:         |         :x:         |  :heavy_check_mark:   |    :heavy_check_mark:    |   :heavy_check_mark:    |       :clock1:       |         :x:         |   :heavy_check_mark:    |        :x:         |
+| [Nexus Group (IN Groupe)](#nexus-group-in-groupe)                       | Certificate Manager    | PKI              | 2024-03-13   |                  :x:                  |                :x:                 |        :x:         |         :x:         |  :clock1:   |    :clock1:    |   :clock1:    |       :x:       |         :x:         |   :x:    |        :x:         |
```
Comment (primetomas) [inline _index.md:70]: Do you need to state the ownership structure (IN Groupe)? It's well visible on Nexus's web page.

File: _index.md (hunk 1 above)
Comment (VDNexus) [reply _index.md]: OK I understand, I made an additional commit to propose an updated version.

File: _index.md (hunk 2 above)
Comment (VDNexus) [reply _index.md]: I made an additional commit to propose an updated version.

**Pull request merged**